internal/store/statefulset_test.go

```diff
@@ -95,6 +97,7 @@ func TestStatefulSetStore(t *testing.T) {
  				kube_statefulset_status_observed_generation{namespace="ns1",statefulset="statefulset1"} 1
  				kube_statefulset_replicas{namespace="ns1",statefulset="statefulset1"} 3
  				kube_statefulset_metadata_generation{namespace="ns1",statefulset="statefulset1"} 3
+ 				kube_statefulset_persistent_volume_claim_retention_policy{namespace="ns1",statefulset="statefulset1",when_deleted="nil",when_scaled="nil"} 1
```

Comment:
If the retention policy is nil it would be better to not expose the metric since it doesn't bring much value and the less timeseries we have to store, the better.

Reply:
done